devel/legacy-support/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        cjones051073 macports-legacy-support 55fdb329676c52cbf3d63f6e886d17d12e32ae3d
+name                legacy-support
+version             20181027
+revision            0
+categories          devel
+platforms           darwin
+maintainers         {jonesc @cjones051073} {kencu @kencu} openmaintainer
+license             GPL-2
```

Comment:
Do we really need GPL for this, no option to go for MIT or something?
Did we take the code from some places where GPL is required?

I'm asking because in many cases this might prevent our ports from being distributable.

Reply:
I'll happily change the license. I just used the same as the snowsleopardfixes port, so it that sense this changes nothing w.r.t. the current situation

Reply:
Actually... No, the license has to be GPL-2 as that is what some of the code @kencu originally used in the showleopardfixes port is licensed as.

Reply:
You are correct, I haven't added a copyright statement to the files I wrote myself. I will add something.

Please point me to where the MIT license is used, as I am not seeing it. I only see the GPL-2 in the files @kencu originally added in the snowleopardfixes port.

Reply:
Copyright statements now added.